### DISCUSSION TITLE
Is it possible to customize the 'bounce rate too high'message?

### DISCUSSION THREAD

**Discussion by brettmoran** [sympa-community/sympa#1700] [Q&A]
Apologies if this is covered somewhere; I can't seem to find it if it has.
I would like to customize this message to include list name and the bounced email address.
Is this possible?

**Comment by racke** [comment]
The template for this message is in mail_tt2/listowner_notification.tt2.

**Comment by brettmoran** [comment]
Thanks!  This helped!
But where listowner_notification.tt2 is processing the bounce_rate type, is the 'who' variable not available?
I've added messaging that references list.name and domain and they work as expected.
However, I've yet to identify a variable that returns the value of the bounce email address.
